src/frontend/efiling-demo/package.json

```diff
@@ -25,7 +25,7 @@
     "react-scripts": "3.4.1",
     "react-test-renderer": "^16.12.0",
     "regenerator-runtime": "^0.13.5",
-    "shared-components": "https://github.com/bcgov/react-shared-components/releases/download/0.4.2/shared-components-0.4.2.tgz",
+    "shared-components": "https://github.com/bcgov/react-shared-components/releases/download/0.4.3/shared-components-0.4.3.tgz",
```

Comment:
need to run `yarn install` after making this change 
i believe there should be a yarn.lock file change happening as well 👍